src/vivarium_gates_nutrition_optimization/constants/data_values.py

```diff
@@ -16,10 +16,12 @@ class _Durations(NamedTuple):
     PARTURITION = 1 * 7
     DETECTION = 6 * 7
     PARTIAL_TERM = 24 * 7
+    INTERVENTION_DELAY = 8 * 7
```

Comment:
Nit: It'd be nice if all these variable names included the units. That would make reading the code easier.

Reply:
done